pom.xml

```diff
@@ -95,21 +95,21 @@
       <artifactId>symphony-api-dal</artifactId>
       <version>${symphonyApiVersion}</version>
       <scope>system</scope>
-      <systemPath>${basedir}/lib/symphony-api-dal-5.3.0.jar</systemPath>
+      <systemPath>${basedir}/lib/symphony-api-dal-5.4.0.jar</systemPath>
```

Comment:
please update httpclient, junit versions, that provoke the dependabot warnings, also please align other dependency versions based on "3rd Party Software - Symphony" confluence page